js/app/widgets/StripePaymentForm.js

```diff
@@ -72,14 +75,23 @@ export default function(form, options) {
   const gatewayForCard = options.card || 'stripe'
   const gatewayConfig = options.gatewayConfigs ? options.gatewayConfigs[gatewayForCard] : { publishableKey: options.publishableKey }
 
-  Object.assign(CreditCard.prototype, stripe)
+  switch (gatewayForCard) {
+    case 'mercadopago':
+      Object.assign(CreditCard.prototype, mercadopago({ onChange: toggleButton }))
```

Comment:
Not sure about overriding `onChange` this way... it's not intuitive, because it means the `onChange` callback passed to the `CreditCard` constructor is never called. 

I agree it's not easy to implement, because `react-payment-inputs` always returns errors, even if the user has not typed anything yet. 

We should find a way to abstract things, so that both Stripe & Mercado Pago behave the same way.

Reply:
I agree. The last line in this snippet shows how the `onChange` function is used in `stripe.js`. Maybe we can send that function from outside that function, then somehow compose the payment gateways in `StripePaymentForm.js` in a similar way. Anyhow, where is `this.config` coming from in **js/app/payment/stripe.js**?

**js/app/payment/stripe.js**
```js
init() {
  this.stripe = Stripe(this.config.gatewayConfig.publishableKey)

  const elements = this.stripe.elements()

  this.card = elements.create('card', { style, hidePostalCode: true })
  this.card.addEventListener('change', this.config.onChange)
},
```